miniircd.py

```diff
@@ -0,0 +1 @@
+miniircd
```

Comment:
This is a symbolic link to the miniircd file, which is needed because `py_modules` in `setup.cfg` only understands `.py` files.